justfile

```diff
@@ -0,0 +1,42 @@
+set dotenv-load
+set ignore-comments
+
+# Default recipe
+[private]
+default:
+    @just --list
+
+# Install necessary tools
+install-tools:
+    rustup component add rustfmt --toolchain nightly
+    cargo install cargo-sort
+
+# Check if necessary tools are installed
+[private]
+check-tools:
+    #!/usr/bin/env bash
+    if ! cargo +nightly fmt --help &> /dev/null; then 
+        echo "cargo +nightly fmt is not installed, use just install-tools or install it manually"
+        exit 1
+    fi
+    if ! cargo sort --help &> /dev/null; then 
+        echo "cargo sort is not installed, use just install-tools or install it manually"
+        exit 1
+    fi
+
+# Format code
+format: check-tools
+    cargo clippy --all-targets --all-features
+    cargo +nightly fmt -- --check
+    cargo sort -w . --check
+
+# Build all binaries and debian packages
+build *ARGS:
+    ci/build_scripts/build.sh {{ARGS}}
```

Comment:
The name `build` is misleading here.

I run it, expecting something along the line of `cargo build`, just to have the surprise to see the script replacing my zig setup with some undesirable pip3 zig. => please check zig install and not pip3zig.

```suggestion
# Release, building all binaries and debian packages
release *ARGS:
    ci/build_scripts/build.sh {{ARGS}}
```

Reply:
Yeah I can make the change in the build.sh (that was my oversight a while back).

Reply:
I added a check for the `zig` binary so it should not destroy your previous zig installation. 2797a7c8abc181a9935b399ef8fc167b88c244e0

Reply:
Thanks